paddle/optimizer/parameter_optimizer_test.cpp

```diff
@@ -0,0 +1,108 @@
+#include "parameter_optimizer.h"
```

Comment:
I remember that WangYi's suggestion that tests should be named with XXX_test.cpp and put it into the same folder.  But I can't find the link....